README.md

```diff
@@ -147,11 +147,11 @@ The relayer is configured via a JSON file, the path to which is passed in via th
 
   `"source-blockchain-id": string`
 
-  - cb58-encoded Blockchain ID of the source Subnet.
+  - cb58-encoded Blockchain ID of the source blockchain.
 
   `"destination-blockchain-id": string`
 
-  - cb58-encoded Blockchain ID of the destination Subnet.
+  - cb58-encoded Blockchain ID of the destination blockchain.
```

Comment:
nit: don't think blockchain needs to be capitalized, and other places they are not.